src/components/videoReco/index.jsx

```diff
@@ -0,0 +1,19 @@
+import { Link } from 'react-router-dom'
+import './videoReco.css'
+
+export function VideoRecommendationCard({video}){
+    const { _id, thumbnail, title, creator, views, likes} = video
+    return(
+        <Link to={`/watch/:${_id}`}>
+            <div className="flex-layout reco-parent" >
+                <img className="reco-image" src={thumbnail} />

```

Comment:
suggestion - some alt attribute can be added here

Reply:
added alt tag